minvws/nl-kat-rocky
# Add task list params

This PR implements a partial scans/tasks lists view, that (according to designs) can be used on: ooi detail page, boefje detail page, and task/scans page. Since, the designs show that the functionality is the same across these pages, it made sense to create one partial and include these in the templates. At the moment in this PR it is implemented on the ooi detail and plugin detail page.

Open questions:

* Since there are different strategies for implementing forms/pagination throughout rocky the main question is if this implementation and location of files etc. is the way forward and correct for the future. With this PR I don't want to introduce yet another new implementation.

* In the designs, the table's first column is the object name. It might not make sense in the context of an ooi detail page, since the name is already present. Additionally, there is a search field for an object name, in the context of a ooi detail this would make not much sense.

* Now form fields are being passed to the template context. This is done to make a differentiation between fields in the filter form fields that need to remain as query parameters when submitting the filter form. This could potentially be a django form, the question is if this is desirable.

How to test/view:
* In mula checkout https://github.com/minvws/nl-kat-mula/pull/16
* Go through the onboarding
* Add an url and do a regular scan
* Choose an object, and scroll down to see the scan history
* Go to kat-alogus
* Select plugin dns-records
